backend/clubs/views.py

```diff
@@ -5018,6 +5021,45 @@ def _calculate_cart_total(cart) -> float:
         )
         return cart_total
 
+    def partial_update(self, request, *args, **kwargs):
+        """
+        Update a ticket's attendance (only accessible by club officers)
+        ---
+        requestBody:
+            content:
+                application/json:
+                    schema:
+                        type: object
+                        properties:
+                            attendance:
+                                type: boolean
+        responses:
+            "200":
+                content:
+                    application/json:
+                        schema:
+                            $ref: '#/components/schemas/Ticket'
+            "400":
+                content:
+                    application/json:
+                        schema:
+                           type: object
+                           properties:
+                                detail:
+                                    type: string
+        ---
+        """
+        attended = request.data.get("attended")
+        if attended is None or not isinstance(attended, bool):
+            return Response(
+                {"detail": "Missing boolean attribute 'attended'."},
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+        ticket = self.get_object()
+        ticket.attended = attended
+        ticket.save()
```

Comment:
Should we allow officers to mark tickets that have been claimed as unclaimed? Don't have strong opinions but would like to hear what you think.

Reply:
I imagine there will be a toggle for attendance for each ticket on the frontend in case officers don't want to scan QR codes, in which case we should allow this

Reply:
Will there ever be a case where a ticket that's been marked as attended needs to be unmarked? If not, let's add a `dry_run` attribute or something. I don't think we should expose functionality that we never want used.